cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -68,12 +68,17 @@
     /sys/fs/cgroup/freezer/snap.*/ w,
     /sys/fs/cgroup/freezer/snap.*/cgroup.procs rw,
 
-    # cgroup: pids
+    # cgroup: snapd
+    # allow creating and configuring the snapd hierarchy
+    /run/snapd/ rw,
+    /run/snapd/cgroup/ rw,
+    mount fstype=cgroup options=(rw nosuid nodev noexec relatime) cgroup -> /run/snapd/cgroup/,
```

Comment:
Use `none -> /run/snapd/cgroup/,` when taking my suggestion, above.

Reply:
I opted not to as explained above.